editor/meta-boxes/index.js

```diff
@@ -10,11 +10,15 @@ import MetaBoxesIframe from './meta-boxes-iframe';
 import MetaBoxesPanel from './meta-boxes-panel';
 import { getMetaBox } from '../selectors';
 
-function MetaBox( { location, isActive } ) {
+function MetaBox( { location, isActive, usePanel = false } ) {
 	if ( ! isActive ) {
 		return null;
 	}
 
+	if ( ! usePanel ) {
```

Comment:
Minor: To avoid the duplication, we could do the reverse and apply the panel as a wrapper:

```jsx

let element = <MetaBoxesIframe location={ location } />;

if ( usePanel ) {
	element = <MetaBoxesPanel>{ element }</MetaBoxesPanel>;
}

return element;
```